tests/test_git.py

```diff
@@ -56,7 +56,7 @@
 
 CMD ["/usr/sbin/sshd", "-De"]
 
-HEALTHCHECK --interval=5s --timeout=5s --retries=5 CMD ["/usr/bin/ssh-keyscan", "-vvv", "-H", "-p", "{_SSH_PORT}", "127.0.0.1"]
+HEALTHCHECK --interval=5s --timeout=5s --retries=5 --start-period=1m CMD ["/usr/bin/ssh-keyscan", "-vvv", "-H", "-p", "{_SSH_PORT}", "127.0.0.1"]
```

Comment:
can you make this ppc64le specific ? I think 1m is unreasonable otherwise.

Reply:
I can, but in this case I thought that we do not care about the start period as a test subject.

Reply:
Amended, but IMHO this looks now really ugly.

Reply:
we do care about startup time because in a kubernetes environment the container will fail to start if it isn't ready within the startup time configured (which by default is rather low)